File: SpringProjects/SpringFinalProject/Frontend/SpringBootFinalProject/src/app/DTOs/appointmentSearch-dto.ts
export class AppointmentSearchDTO {

  // category: number;
  // hospital: number;
  // doctor: number;
  // date: string;

  constructor(private client: number, private doctor: number, private date: Date, private status: string) {
    // this.cat = cat;
    // this.hospital = hos;
    // this.doctor = doc;
    // this.date = d;
  }

}

File: SpringProjects/SpringFinalProject/Backend/AppointmentService/src/main/java/lk/e_channelling/appointment_service/dto/AppointmentSearchDto.java
package lk.e_channelling.appointment_service.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.Instant;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class AppointmentSearchDto {

    private Integer client;
    private Integer doctor;
    private Instant date;
    private String status;

}
